assets/src/edit-story/app/story/storyProvider.js

```diff
@@ -154,7 +162,7 @@ function StoryProvider({ storyId, children }) {
       animationState,
       capabilities,
       meta: {
-        isSaving: isSaving || isAutoSaving,
+        isSaving: isSaving || isAutoSaving || isSavingMetaBoxes,
```

Comment:
This is pretty cool:

The user does not realize we're first saving the story and then the meta boxes. They just see one continuous saving animation.

Reply:
In my testing, saving a story now takes around 5 seconds. Is this normal? As much as I think this is cool, I think we should maybe proritize the story saving. I think that is what important, if the meta box fails to save, then so be it.

Reply:
Note that saving the story itself is not impacted, it's just that the saving _animation_ is displayed a bit longer.